crux-bench/src/crux/bench/watdiv_datomic.clj

```diff
@@ -351,11 +351,12 @@
                                                                             :args [(d/db conn)]}))}))))))))))))
 
 (defn -main []
-  (let [output-file (io/file "datomic-results.edn")]
-    (bench/save-to-file output-file
-                        (->> (run-watdiv-bench {:test-count 100})
-                             (filter :query-idx)
-                             (sort-by :query-idx)))
+  (let [output-file (io/file "datomic-results.edn")
+        watdiv-results (run-watdiv-bench {:test-count 100})]
+    (bench/save-to-file output-file (conj (->> (rest watdiv-results)
```

Comment:
nit: this is correct, but when I was reading it I had to double-check whether you wanted to put the `(first watdiv-results)` on the end or the beginning of this seq. let's take away the ambiguity with `(cons (first ...) (->> (rest ...) ...))`?